FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -2074,17 +2095,26 @@ subroutine GFS_physics_driver                                 &
 
       do i=1,im
         Diag%epi(i)     = ep1d(i)
-        Diag%dlwsfci(i) = adjsfcdlw(i)
+        Diag%dlwsfci(i) = adjsfcdlw_for_lsm(i)
         Diag%ulwsfci(i) = adjsfculw(i)
-        Diag%uswsfci(i) = adjsfcdsw(i) - adjsfcnsw(i)
-        Diag%dswsfci(i) = adjsfcdsw(i)
+        Diag%uswsfci(i) = adjsfcdsw_for_lsm(i) - adjsfcnsw_for_lsm(i)
+        Diag%dswsfci(i) = adjsfcdsw_for_lsm(i)
         Diag%gfluxi(i)  = gflx(i)
         Diag%t1(i)      = Statein%tgrs(i,1)
         Diag%q1(i)      = Statein%qgrs(i,1,1)
         Diag%u1(i)      = Statein%ugrs(i,1)
         Diag%v1(i)      = Statein%vgrs(i,1)
       enddo
 
+      if (Model%override_surface_radiative_fluxes) then
+        do i=1,im
+          Diag%dlwsfci_rrtmg(i) = adjsfcdlw(i)
+          Diag%ulwsfci_rrtmg(i) = adjsfculw(i)
+          Diag%uswsfci_rrtmg(i) = adjsfcdsw(i) - adjsfcnsw(i)
+          Diag%dswsfci_rrtmg(i) = adjsfcdsw(i)
```

Comment:
This sets them back to what they were before this module?

Reply:
It sets them to what they were predicted to be by the fortran radiation code.